manifest_staging/charts/gatekeeper/templates/gatekeeper-controller-manager-deployment.yaml

```diff
@@ -24,6 +24,10 @@ spec:
       heritage: '{{ .Release.Service }}'
       release: '{{ .Release.Name }}'
   strategy:
+    {{- if .Values.controllerManager.strategyRollingUpdate }}
```

Comment:
Thanks for the PR!

Can you pls add this to values.yaml and the README in https://github.com/open-policy-agent/gatekeeper/tree/master/cmd/build/helmify/static then run `make manifests`

Reply:
thank you, I've updated the PR accordingly